packages/runtime-core/src/vnode.ts

```diff
@@ -214,17 +214,7 @@ function trackDynamicNode(vnode: VNode) {
 export function cloneVNode(vnode: VNode): VNode {
   return {
     _isVNode: true,
-    type: vnode.type,
-    props: vnode.props,
-    key: vnode.key,
-    ref: vnode.ref,
-    children: vnode.children,
-    target: vnode.target,
-    shapeFlag: vnode.shapeFlag,
-    patchFlag: vnode.patchFlag,
-    dynamicProps: vnode.dynamicProps,
-    dynamicChildren: vnode.dynamicChildren,
-    appContext: vnode.appContext,
+    ...vnode,
```

Comment:
This is intentional to avoid the overhead of the spread.

Reply:
I revert this change and now this PR is all about tests